### PR TITLE
Update scenario indentation

### PR DESCRIPTION
**Pull request by nanux**

### CODE REVIEW

File: taskcat/stacker.py [hunk 1]
```diff
@@ -1400,7 +1400,7 @@ def validate_yaml(self, yaml_file):
         ]
         try:
             if os.path.isfile(yaml_file):
-                print(self.nametag + " :Reading Config from: {0}".format(yaml_file))
+                print(self.nametag + " :Reading Config form: {0}".format(yaml_file))
```
Comment (villasv) [inline taskcat/stacker.py]: Is this change intentional? What’s the reasoning behind it?

Comment (avattathil) [reply taskcat/stacker.py]: That was a typo. I will correct it

Comment (nanux) [reply taskcat/stacker.py]: I am really sorry, I have no idea how that happened. I have edited the file on Github and I remember having only one file in the PR.
